src/sqlfluff/core/parser/segments/base.py

```diff
@@ -903,7 +902,10 @@ def recursive_crawl(self, *seg_type, recurse_into=True):
         if recurse_into or not match:
             # Recurse
             for seg in self.segments:
-                yield from seg.recursive_crawl(*seg_type, recurse_into=recurse_into)
+                yield from seg.recursive_crawl(
+                    *seg_type,
+                    recurse_into=recurse_into,
+                )
 
```

Comment:
Do we need this change?

Reply:
Nope. Seems like Black is not behaving like I'd expect -- remove the original change, it should go back to one line. 🤔 I'll restore it manually.